independent-publisher-connectors/myStrom/apiDefinition.swagger.json

```diff
@@ -0,0 +1,640 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "myStrom",
+    "description": "myStrom is a Swiss company producing smart home devices. With this connector it is possible to control myStrom devices over the REST API.",
+    "version": "1.0",
+    "contact": {
+      "name": "Tomasz Poszytek",
+      "url": "https://aka.ms/poszytek",
+      "email": "tomasz@poszytek.eu"
+    }
+  },
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://mystrom.ch/"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://mystrom.ch/privacy-policy/"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Internet of Things;Lifestyle and Entertainment"
+    }
+  ],  
+  "host": "mystrom.ch",
+  "basePath": "/api/v2/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": ["application/json"],
+  "produces": ["application/json"],
+  "paths": {
+    "/auth": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "Response",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "authToken": {
+                  "type": "string",
+                  "description": "Token to be used to authenticate other requests.",
+                  "title": "Authentication token"
+                }
+              }
+            }
+          },
+          "default": {
+            "description": "Default response",
+            "schema": {}
```

Comment:
@tposzytek Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide valid response schema definition for all the operations in the swagger.

Reply:
Done